server/app/views/applicant/ProgramCardViewRenderer.java

```diff
@@ -219,42 +217,6 @@ public LiTag createProgramCard(
     return cardListItem.with(programData).with(actionDiv);
   }
 
-  private static Optional<ImgTag> createProgramImage(
-      Http.Request request,
-      ProgramDefinition program,
-      Locale preferredLocale,
-      SettingsManifest settingsManifest,
-      PublicStorageClient publicStorageClient) {
-    if (!settingsManifest.getProgramCardImages(request)) {
-      return Optional.empty();
-    }
-    if (program.summaryImageFileKey().isEmpty()) {
-      return Optional.empty();
-    }
-    String summaryImageFileKey = program.summaryImageFileKey().get();
-    if (!PublicFileNameFormatter.isFileKeyForPublicProgramImage(summaryImageFileKey)) {
-      return Optional.empty();
-    }
-    // TODO(#5676): Can we detect if the image URL is invalid and then not show it?
```

Comment:
I've removed these TODOs in `ProgramImageUtils` because after talking with Bion, we decided that better handling for loading/error states isn't worth the implementation time and future maintenance cost.